docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.5'
+
+volumes:
+  mysql-db:
+
+services: 
+
+  mysql:
+    image: mysql:latest
```

Comment:
You may have problems if the next version has any breaking changes. I think its better set the version, what you think?

Reply:
I agree. I adjusted this.